images/gradle/config/main.tf

```diff
@@ -0,0 +1,28 @@
+terraform {
+  required_providers {
+    apko = { source = "chainguard-dev/apko" }
+  }
+}
+
+variable "extra_packages" {
+  description = "The additional packages to install (e.g. gradle-8, openjdk-17)."
+  default     = []
+}
+
+variable "java_home" {
+  description = "The JAVA_HOME environment variable."
+  default     = "/usr/lib/jvm/java-17-openjdk"
+}
+
+data "apko_config" "this" {
+  config_contents = file("${path.module}/template.apko.yaml")
+  extra_packages  = var.extra_packages
+}
+
+output "config" {
+  value = jsonencode(merge(data.apko_config.this.config, {
+    environment : {
+      "JAVA_HOME" : var.java_home
```

Comment:
JAVA_HOME is still defined in template.apko.yaml on line 24
keep one of them?

Reply:
Good catch, removed from the YAML